src/main/java/no/nav/tag/tiltaksgjennomforing/avtale/AvtaleController.java

```diff
@@ -363,6 +372,7 @@ public void godkjenn(
     ) {
         Avtalepart avtalepart = innloggingService.hentAvtalepart(innloggetPart);
         Avtale avtale = avtaleRepository.findById(avtaleId).orElseThrow(RessursFinnesIkkeException::new);
+        sjekkArbeidstreningToggle(avtale);
```

Comment:
Jeg ville kanskje gjort:

```suggestion
        Avtale avtale = avtaleRepository.findById(avtaleId)
            .map(this::sjekkArbeidstreningToggle)
            .orElseThrow(RessursFinnesIkkeException::new);
```